cmd/fuego/commands/controller.go

```diff
@@ -25,19 +25,40 @@ func Controller() *cli.Command {
 				fmt.Println("Note: You can add a controller name as an argument. Example: `fuego controller books`")
 			}
 
-			_, err := createController(controllerName)
+			_, err := createControllerFile(controllerName, "controller.go", controllerName+".go")
 			if err != nil {
 				return err
 			}
 
 			fmt.Printf("🔥 Controller %s created successfully\n", controllerName)
 			return nil
 		},
+		Flags: []cli.Flag{
```

Comment:
I prefer if you make a function `createService` that is either called by `fuego service` or additionally by `fuego controller --with-service`.

You can access the flag within cli context by calling `cCtx.String("with-service")` in the Action.